roles/login_ocp/tasks/main.yml

```diff
@@ -6,4 +6,4 @@
 - name: login to cluster
   vars:
     kubeadmin_password: "{{ ansible_env.kubeadmin_password }}"
-  shell: oc login -u admin -p {{ kubeadmin_password }}
+  shell: oc  -u {{ login }} -p {{ kubeadmin_password }}
```

Comment:
Will this work? Isn't this `oc -u <user> -p <password>` without `login`?

Reply:
somehow missed this, it's been a long week :)